certs/user.go

```diff
@@ -0,0 +1,74 @@
+package certs
+
+import (
+	"crypto"
+	"crypto/rand"
+	"crypto/rsa"
+	"encoding/json"
+	"io/ioutil"
+
+	"github.com/nknorg/nkn/v2/node"
+	"github.com/nknorg/nkn/v2/util/config"
+
+	"github.com/go-acme/lego/v3/registration"
+)
+
+// You'll need a User or account type that implements acme.User
+type User struct {
+	Domain       string
+	Email        string
+	Registration *registration.Resource
+	Key          *rsa.PrivateKey
+}
+
+func (u *User) GetEmail() string {
+	return u.Email
+}
+func (u User) GetRegistration() *registration.Resource {
+	return u.Registration
+}
+func (u *User) GetPrivateKey() crypto.PrivateKey {
+	return u.Key
+}
+func (u *User) SaveToDisk() error {
```

Comment:
Why do we want to save it to disk? node IP might change at any time, so it's better to get the latest one each time nknd starts, not read the previous one